content/applications/sales/crm/track_leads/lead_distribution_report.rst

```diff
@@ -0,0 +1,205 @@
+========================
+Lead distribution report
+========================
+
+A *lead distribution report* can be used to see if active leads are being assigned equitably
+across sales members. It can also be used to view the distribution of good or :doc:`quality leads
+<quality_leads_report>`, and see how frequently each salesperson is receiving (and keeping) leads.
+
+Lead distribution reports can be run each week to help keep salespeople on track, while
+providing them with ample good leads. These reports can also be used to see whether sales members
+are staying productive, if good leads are being lost too often by one salesperson, and what
+percentage of good leads are being retained overall.
+
+Create lead distribution reports
+================================
+
+To create a lead distribution report, first navigate to :menuselection:`CRM app --> Reporting -->
+Pipeline`, which reveals the :guilabel:`Pipeline Analysis` dashboard.
+
+Remove all the default filters in the search bar at the top of the page. Doing so
+displays data related to *all* leads.
+
+Custom filters can now be added by clicking the :icon:`fa-caret-down` :guilabel:`(down caret)`
+icon, to the right of the search bar, to reveal a drop-down menu of search and filter
+options.
```

Comment:
Suggestion to add references to these sections, not mandatory but could be useful :)
```suggestion
:ref:`Custom filters <search/custom-filters>` can now be added by clicking the :icon:`fa-caret-down`
:guilabel:`(down caret)` icon, to the right of the search bar, to reveal a drop-down menu of search
and filter options.
```